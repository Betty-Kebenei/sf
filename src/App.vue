<template>
  <div id="app">
    <Navbar />
    <HelloWorld msg="Welcome to Your Vue.js App" />
  </div>
</template>;

<script>
import HelloWorld from './components/HelloWorld.vue';
import Navbar from './common/Navbar.vue';

export default {
  name: 'App',
  components: {
    HelloWorld,
    Navbar,
  },
};
</script>

<style lang="scss" >
   @import './assets/scss/App.scss'
</style>
