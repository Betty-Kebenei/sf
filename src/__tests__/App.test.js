import { shallowMount } from '@vue/test-utils';
import App from '../App.vue';

describe('App', () => {
  const wrapper = shallowMount(App);
  test('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
