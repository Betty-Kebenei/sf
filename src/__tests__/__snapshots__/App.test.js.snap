// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  id="app"
>
  <img
    alt="Vue logo"
    src="./assets/logo.png"
  />
   
  <helloworld-stub
    msg="Welcome to Your Vue.js App"
  />
</div>
`;
